recipes/liburing/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package C)
 
+project(test_package C)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-find_package(liburing REQUIRED CONFIG)
```

Comment:
Revert. Cmake package multi should be used by default: https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#cmake-targets